packages/website/src/components/lib/markdown.ts

```diff
@@ -54,7 +54,7 @@ export function createMarkdown(state: ConfigModel): string {
  */
 export function createMarkdownParams(state: ConfigModel): string {
   const { rules } = parseESLintRC(state.eslintrc);
-  const ruleKeys = Object.keys(rules);
+  const ruleKeys = Object.keys(rules ?? {});
```

Comment:
Oh, I guess you'll have to fix the lint error that complains about this not being possibly-nullish